terraform/clusters/pauld.run-sandbox.aws.ext.govsvc.uk/cluster.tf

```diff
@@ -0,0 +1,67 @@
+terraform {
+  backend "s3" {
+    bucket = "gds-re-run-sandbox-terraform-state"
+    region = "eu-west-2"
+    key    = "pauld.run-sandbox.aws.ext.govsvc.uk/cluster.tfstate"
+  }
+}
+
+data "aws_caller_identity" "current" {}
+
+module "gsp-cluster" {
+    source = "git::https://github.com/alphagov/gsp-terraform-ignition//modules/gsp-cluster"
+    cluster_name = "pauld"
+    dns_zone = "run-sandbox.aws.ext.govsvc.uk"
+    user_data_bucket_name = "gds-re-run-sandbox-terraform-state"
+    user_data_bucket_region = "eu-west-2"
+    k8s_tag = "v1.12.2"
+    admin_role_arns = ["arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/admin"]
+
+    addons = {
+      ingress = 1
+      canary = 0
```

Comment:
Would you enable the canary? It's not strictly required, but is the only mean of monitoring for our clusters.